chapter-1/bins/chap1.c

```diff
@@ -1,6 +1,7 @@
 #include <stdio.h>
 #include <stdlib.h>
-#include "parse.h"
+
+#include <parse.h>
```

Comment:
What's the diff in this context between using quotes or angle brackets?

Reply:
In this context, nothing. I usually treat it as a sign of intent, where the formal interface is in `.h` files in the `include/` directory, and then included with `<>`, and local implementation oriented headers can be put in `src/` with includes using quotes.

We might want to put a directory in to make that clear, e.g.

```cpp
#include <lisp-wasm/parse.h>
```

Ref: https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html